### DISCUSSION TITLE
Sound Notification Change

### DISCUSSION THREAD

**Discussion by NetSorfcusu** [3dproger/AxelChat#580] [Ideas]
It would be nice to be able to customize the notification sound when a message arrives in the app. The current sound is very annoying and cannot be changed. Please add an option to change the notification sound.

**Comment by 3dproger** [comment]
Hi! Sorry for not responding for so long. I will try to make this setting as soon as I have time

**Comment by 3dproger** [comment]
Hi! Now in the latest update you can simply replace file `<app_folder>/deps/audio/notifications/default1.wav` with your own. This way, you can use any sound instead of the default one. Only the wav format is supported, and after changing the file, you'll need to restart AxelChat if it's running. I hope to be able to implement more convenient settings someday
